standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -5841,9 +5851,7 @@ private <T> T doFetchProperties(String key, java.util.function.Function<MMetasto
         return (T) (transform != null? transform.apply(properties) : properties);
       }
     } finally {
-      if (query != null) {
-        query.closeAll();
```

Comment:
nit: I'd prefer `query.closeAll()` other than `query.close()`, quoted from `Query` class

```
Don't use this method directly; use closeAll() instead. It is intended for use with try-with-resources.
Throws:
Exception – if this resource cannot be closed
```
If you want to use with try-with-resources, `QueryWrapper` is a choice.